MdePkg/Include/Library/PanicLib.h

```diff
@@ -0,0 +1,74 @@
+/** @file

```

Comment:
The header should describe what "panic" means and the cases when panic APIs should be used.

Reply:
Updated.